script/jenkins-utilities

```diff
@@ -139,13 +139,12 @@ __EOF__
 # Creating and activating a service
 function create_service() {
   local esp_service="${1}"
-  local swagger_json="${2}"
   local output=''
   local operation=''
   local version=''
 
   echo 'Deploying service'
-  retry -n 3 run ${GCLOUD} endpoints services deploy "${swagger_json}"
+  retry -n 3 run ${GCLOUD} endpoints services deploy ${@:2}
```

Comment:
is this change necessary?

Reply:
Yes.  to pass multiple config files.   for grpc, you need to pass descriptor file in addition to yaml file